mujinvisioncontrollerclient/visioncontrollerclient.py

```diff
@@ -399,110 +422,241 @@ def StartVisualizePointCloudTask(self, vminitparams, locationName=None, sensorSe
 
     def BackupVisionLog(self, cycleIndex, sensorTimestamps=None, fireandforget=False, timeout=2.0):
         # type: (str, Optional[List], bool, float) -> Optional[Dict]
-        if sensorTimestamps is None:
-            sensorTimestamps = []
-        command = {'command': 'BackupDetectionLogs', 'cycleIndex': cycleIndex, 'sensorTimestamps': sensorTimestamps}
+        """Backs up the vision log for a given cycle index
+
+        Args:
+            cycleIndex (str): Unique cycle index string for tracking, backing up, and differentiating cycles.
```

Comment:
I checked `mujinvisionmanager.cpp` cycleindex can be an empty string, so it could be optional (or empty string) in the client.

Reply:
Noted. Let's leave that as a ToDo for now though.